app/jobs/enque_parallel_institution_create_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class EnqueParallelInstitutionCreateJob < ApplicationJob
+  queue_as :default
+
+  def perform(*_args)
+    if Institution.count != 0
+      Rails.logger.debug 'To execute this job there can be no Institution records in the database'
+      return
+    end
+
+    relevant_institution_types_ids = [93, 94, 45, 96, 14, 15, 17, 15, 27, 26, 20, 16]
```

Comment:
These seem like some magic numbers to me. Would be great to comment them somehow. I suggest assigning them to named consts - that way it'll be very easy to understand what they mean and to reuse them later if needed. Also, comments or documentation tends to get outdated very easily, while named variables do not, and so we end with self-documenting code. Something like
```ruby
PRIMARY_SCHOOL = 1
ANOTHER_ASYLUM = 2
...
```